arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -41,7 +39,7 @@
         "tags": [
           "Namespaces"
         ],
-        "operationId": "Namespaces_ListBySubscription",
+        "operationId": "Namespaces_List",
```

Comment:
Also, just to clarify we will still keep the Client "version": "2015-08-01" in the new swagger.

"info": {
    "title": "ServiceBusManagementClient",
    "description": "Azure Service Bus client",
    "version": "2015-08-01"
  },